frameworks.md

```diff
@@ -8,8 +8,7 @@ we can have empty tests.
 
 Tests are grouped in `suites`, which can also be `nested`. Tests can be placed also outside of a suite, we call them *global tests*.
 
-Internally Mocha wraps everything in a suite, we call it *global suite*, so the global tests will become the tests of the aformentioned 
-suite, as also all other top level suites will become its direct child suites, implicitly all other suites will become its more deeper child suites, in a recursive structure. This is great for our CRI, because it is really what we need 
+Internally Mocha wraps everything in a suite, we call it *global suite*, so the global tests will become the tests of the aformentioned suite, as also all other top level suites will become its direct child suites, implicitly all other suites will become its more deeper child suites, in a recursive structure. This is great for our CRI, because it is really what we need
```

Comment:
This still talks about CRI (I don't think it should), and the linebreak at the end of the line is unncessary.